tests/datasets/api_tests.py

```diff
@@ -189,10 +190,9 @@ def pg_test_query_parameter(query_parameter, expected_response):
                 "admin_database",
                 "information_schema",
                 "public",
-                "superset",
             ]
             expected_response = {
-                "count": 5,
+                "count": 4,
```

Comment:
This should be replaced by an actual db.query instead of the hardcoded value

Reply:
But then if there's a regression in reading from the DB the unit test wouldn't capture it, since the expected value and the API response are both built from the DB call. Wouldn't it be better to leave the hardcoded data to detect that?

Reply:
Good point, I've done the same, but I'm wondering if it's the actual count from what's on the DB if it would be better for us to change this? (non blocking for sure)